src/PrestaShopBundle/Form/Admin/Catalog/Category/AbstractCategoryType.php

```diff
@@ -186,8 +186,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'label' => $this->trans('Category thumbnail', 'Admin.Catalog.Feature'),
                 'help' => $this->trans('It will display a thumbnail on the parent category\'s page, if the theme allows it.', 'Admin.Catalog.Help'),
                 'required' => false,
-                'can_be_deleted' => false,
+                'can_be_deleted' => true,
                 'show_size' => true,
+                'csrf_delete_token_id' => 'delete-thumbnail-image',
```

Comment:
Why do you use custom CSRF token?

Reply:
Because category cover deletion mechanism uses it also.

Reply:
Done :-)